data-es6.js

```diff
@@ -3893,6 +3941,54 @@ exports.tests = [
         iojs:        true,
       },
     },
+    'constructor accepts null': {
```

Comment:
Please add `es6shim: true` to this test, since the `es6-shim`'s `Set` constructor does accept null